foxy/distribution.yaml

```diff
@@ -2836,11 +2836,10 @@ repositories:
       - ros2_control
       - ros2controlcli
       - test_robot_hardware
-      - transmission_interface
       tags:
         release: release/foxy/{package}/{version}
       url: https://github.com/bmagyar/ros2_control-release.git
```

Comment:
@bmagyar same question as in #27834 about different release repository going from Dashing to Foxy.

Reply:
https://github.com/ros/rosdistro/pull/27873